cli/src/swam/cli/Main.scala

```diff
@@ -318,7 +328,38 @@ object Main extends CommandIOApp(name = "swam-cli", header = "Swam from the comm
                 res <- engine.validate(module).attempt
                 _ <- res.fold(t => logger.error("Module is invalid", t), _ => logger.info("Module is valid"))
               } yield ExitCode.Success
+            case Infer(file, wat, wasi, func_name) =>
+              for {
+                engine <- Engine[IO](blocker)
+                tcompiler <- swam.text.Compiler[IO](blocker)
+                module = if (wat) tcompiler.stream(file, false, blocker) else engine.sections(file, blocker)
 
+                compiled <- engine.compile(module)
+                _ <- IO(compiled.functions.zipWithIndex.map {
+                        case (CompiledFunction(typeIndex, tpe, locals, code), fidx) => 
+                          val functionName =
+                          compiled.names.flatMap(_.subsections.collectFirstSome {
+                            case FunctionNames(names) =>
+                              names.get(typeIndex)
+                            case _ =>
+                              None
+                        })
+                        functionName match {
+                          case Some(x) => {
+                            if(x == func_name){
+                              println(s"This is function name : $func_name")
+                              println(s"This is param names : ${tpe.params}")
```

Comment:
To better integrate with the already implemented features:
- [ ] Remove the prefix text "This is a param names: ..."
- [ ] Map the SWAM types to the ones that we are using in the env for wafl: `I32` to `Int32`, `I64` to `Int64`, `F32` to `Float32` and `F64` to `Float64`
- [ ] Print only the data types separated by comma as follows: 'Int32,Int64...'

Reply:
> To better integrate with the already implemented features:
> 
> * [x]  Remove the prefix text "This is a param names: ..."
> * [x]  Map the SWAM types to the ones that we are using in the env for wafl: `I32` to `Int32`, `I64` to `Int64`, `F32` to `Float32` and `F64` to `Float64`
> * [x]  Print only the data types separated by comma as follows: 'Int32,Int64...'

Printing the parameters below format,

sudo mill cli.run infer --wasi code_analysis/test/resources/coverage-test/formal_data/Deconvolution-1D.wasm _fft

Int32,Int32,Int32,Int32